lib/_debugger.js

```diff
@@ -126,7 +130,7 @@ Protocol.prototype.execute = function(d) {
       break;
 
     default:
-      throw new Error('Unknown state');
+      throw I18N.Error(I18N.NODE_DEBUGGER_UNKNOWN_STATE);
```

Comment:
Yeah, I'm not yet entirely sure if this is the approach I'll settle on. I may put the `new` back in here.